gradle/dist.gradle

```diff
@@ -0,0 +1,89 @@
+import java.nio.file.Paths
+
+import static groovy.io.FileType.FILES
+
+def licenses = [
+        [name: 'avalon-framework', title: 'Avalon Framework', license: ['apache2.txt']],
```

Comment:
* `name`: Maven name or common prefix for this library. It's used to detect we have a license definition for each JAR we bundle.
* `title`: Name of the library
* `license`: Array of license and notices files.